aggregator/rpc_server_test.go

```diff
@@ -73,15 +76,42 @@ func TestProcessSignedStateRootUpdateMessage(t *testing.T) {
 	mockMessageBlsAggServ.EXPECT().ProcessNewSignature(context.Background(), messageDigest,
 		&signedMessage.BlsSignature, signedMessage.OperatorId)
 	mockMessageBlsAggServ.EXPECT().InitializeMessageIfNotExists(messageDigest, coretypes.QUORUM_NUMBERS, []eigentypes.QuorumThresholdPercentage{types.MESSAGE_AGGREGATION_QUORUM_THRESHOLD}, types.MESSAGE_TTL, types.MESSAGE_BLS_AGGREGATION_TIMEOUT, uint64(0))
+	mockOperatorRegistrationsServ.EXPECT().GetOperatorPubkeysById(context.Background(), signedMessage.OperatorId).Return(MOCK_OPERATOR_PUBKEYS, true)
+
 	err = aggregator.ProcessSignedStateRootUpdateMessage(signedMessage, nil)
 	assert.Nil(t, err)
 }
 
+func TestProcessInvalidSignedStateRootUpdateMessage(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	aggregator, _, _, _, _, _, mockOperatorRegistrationsServ, _, _, _, err := createMockAggregator(mockCtrl, MOCK_OPERATOR_PUBKEY_DICT)
+	assert.Nil(t, err)
+
+	message := messages.StateRootUpdateMessage{
+		RollupId:            1,
+		BlockHeight:         2,
+		Timestamp:           3,
+		NearDaCommitment:    keccak256(4),
+		NearDaTransactionId: keccak256(5),
+		StateRoot:           keccak256(6),
+	}
+
+	signedMessage, err := createMockSignedStateRootUpdateMessage(message, *MOCK_OPERATOR_KEYPAIR)
+	assert.Nil(t, err)
+	invalidateSignature(&signedMessage.BlsSignature)
```

Comment:
Could this just reassign the signature to 0? Wasn't totally clear to me when reading what invalidating a signature meant

Reply:
Yeah, exactly. It just feels like it's better to try with a signature we know is invalid as opposed to calling something - doing it like that just feels like it depends on some side effect

Reply:
Not sure what you mean its better. Negating valid signature - produces invalid one.

Reply:
I aggree with the comment but I would do it differently:

```suggestion
	signedMessage.BlsSignature = NewInvalidSingature()
```

Passing a field by reference to mutate it is what makes it confusing. Either way is just an opinion

Reply:
Done in [bc67968](https://github.com/NethermindEth/near-sffl/pull/194/commits/bc679685e81a3f20ebf09d443395b20dfb5def13)